code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,418 @@
+		to_chat(user, span_notice("The thicket crawls through the frame, and you suddenly find vines beneath you..."))
+
+
+
+// Lady out of gates, gives a brain trauma causing the person to scratch themselves
+/obj/item/wallframe/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her..."
+	icon_state = "eldritch_painting_beauty"
+	result_path = /obj/structure/sign/painting/eldritch/beauty
+
+/obj/structure/sign/painting/eldritch/beauty
+	name = "Lady out of gates"
+	desc = "A painting depicting a perfect lady, and I must be perfect like her. Destroyable with wirecutters."
+	icon_state = "eldritch_painting_beauty"
+	sensor_type = /datum/proximity_monitor/advanced/eldritch_painting/beauty
+	// Set to mutadone by default to remove mutations
+	var/list/reagents_to_add = list(/datum/reagent/medicine/mutadone)
+
+// The special examine interaction for this painting
+/obj/structure/sign/painting/eldritch/beauty/examine(mob/living/carbon/human/user)
+	if(IS_HERETIC(user))
+		to_chat(user, "Your imperfections shed and you are restored.")
+		user.reagents.add_reagent(reagents_to_add, 5)
+	if(!user.has_dna())
+		return
+	if(user.has_trauma_type(/datum/brain_trauma/severe/eldritch_beauty))
+		to_chat(user, "You feel changed, more perfect....")
+		user.easy_random_mutate(NEGATIVE + MINOR_NEGATIVE)
+
+
+// Specific proximity monitor for Lady out of gates or /obj/item/wallframe/painting/eldritch/beauty
+/datum/proximity_monitor/advanced/eldritch_painting/beauty
+	applied_trauma = /datum/brain_trauma/severe/eldritch_beauty
+	text_to_display = "Her flesh glows in the pale light, and mine can too...If it wasnt for these imperfections...."
+
+/datum/brain_trauma/severe/eldritch_beauty
+	name = "The Pursuit of perfection"
+	desc = "Patient seems to furiously scratch at their body, the only way to make them cease is for them to remove their jumpsuit."
+	scan_desc = "I_)8(P_E##R&&F(E)C__T)"
+	gain_text = span_warning("I WILL RID MY FLESH FROM IMPERFECTION!! I WILL BE PERFECT WITHOUT MY SUITS!!")
+	lose_text = span_notice("You feel the influence of something slip your mind, and you feel content as you are.")
+	/// How much damage we deal with each scratch
+	var/scratch_damage = 0.5
+
+/datum/brain_trauma/severe/eldritch_beauty/on_life(seconds_per_tick, times_fired)
+	// If they don't have a jumpsuit, return. They are encouraged to remove their jumpsuit to avoid damage.
+	if(!owner.get_item_by_slot(ITEM_SLOT_ICLOTHING))
+		return
+
+	// Scratching code
+	var/obj/item/bodypart/bodypart = owner.get_bodypart(owner.get_random_valid_zone(even_weights = TRUE))
+	if(bodypart && IS_ORGANIC_LIMB(bodypart) && !(bodypart.bodypart_flags & BODYPART_PSEUDOPART))
+		if(!owner.incapacitated())
+			bodypart.receive_damage(scratch_damage)
+			if(prob(2))
+				to_chat(owner, span_notice("You scratch furiously at [bodypart] to rid its imperfections!"))
```

Comment:
Early returns on this would be appreciated

Reply:
Yeah, It should work like that now.